src/control/cmd/daos/pool.go

```diff
@@ -92,7 +92,9 @@ func (cmd *poolBaseCmd) connectPool() error {
 	case cmd.PoolID().HasUUID():
 		cmd.poolUUID = cmd.PoolID().UUID
 		cmd.log.Debugf("connecting to pool: %s", cmd.poolUUID)
-		rc = C.daos_pool_connect(cmd.poolUUIDPtr(), cSysName,
+		cUUIDstr := C.CString(cmd.poolUUID.String())
```

Comment:
@mjmac Could you please advise whether this is the correct way to extract the UUID string?